README.md

```diff
@@ -849,6 +851,101 @@ The authentication resource allows a user to authenticate a given reader's passw
+
+#### Using a Token
+
+Given a User ID, the token resources generate a token value that, when used in combination with the ID, will authenticate a reader.
+Currently, these token values can only be passed directly to an edition by means of the edition's URL.
+By inserting both the ID and the token value into the URL as query parameters, the edition can be authenticated without requiring interaction by the reader.
+
+In order to improve the seamless experience for your users, when a token generated by this API is used to authenticate a reader, the server will automatically provide that reader with another token.
+This new token will continue to authenticate the reader for future visits for the rest of that day.
+This token will be stored in the browser's local storage and so will be available for any visit from the same device and browser.
+However, should token details be provided in the URL your reader follows, these will take precedence over any locally-stored token.
+
+Note that since these tokens have a limited lifetime, if a user does attempt to reuse a URL with a token after the token has expired, they may be presented with a login screen as other users would be.
+If the intention is to provide users with a seamless experience, then fresh URLs may need to be generated for them frequently, and they should be alerted to the limited lifetime of the URLs thus generated.
+A token-less URL may be intentionally provided to allow users to use locally-stored token details.
+
+##### Token URL Query Parameters
+
+To successfully authenticate an edition using the token details, the following query parameters should be specified:
+
+| Query Parameter Name | Token Parameter Name | Description                                                    |
+| -------------------- | -------------------- | -------------------------------------------------------------- |
+| `yuduAuthId`         | `key`                | The unique User ID for whom the token was generated            |
+| `yuduAuthToken`      | `tokenValue`         | The generated value of the token returned in the response body |
+
+For example, if your edition URL is `http://hosted.edition.domain/path/to/edition/index.html` then the token above could be used by directing the user to the destination `http://hosted.edition.domain/path/to/edition/index.html?yuduAuthId=uniqueUserIdentification&yuduAuthToken=0123456789abcdefghijklmnopqrstu`.
+A simple use-case could be as follows:
+
+1. A reader clicks on a link on your webpage indicating they wish to view an edition.
+2. Your server reacts to that request by:
+    1. sending a request to this API for a token for that user
+    2. retrieving the token value from the response
+    3. inserting the token value and user ID into the edition's target URL
+    4. returning a 303 redirect with the modified edition URI as the target
+3. The reader's browser redirects to the edition and the edition uses the token to authenticate.
+
```

Comment:
Think so. It should be fine for anyone with good developers, and for anyone who needs more than this we'll probably need an entire reference implementation - which can wait until we have to create one.